[2021-12-15 04:57:52,229] {processor.py:163} INFO - Started process (PID=63859) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 04:57:52,229] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py for tasks to queue
[2021-12-15 04:57:52,230] {logging_mixin.py:109} INFO - [2021-12-15 04:57:52,230] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 04:57:52,253] {processor.py:632} INFO - DAG(s) dict_keys(['tutorial_taskflow_api_etl']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 04:57:52,257] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 04:57:52,510] {logging_mixin.py:109} INFO - [2021-12-15 04:57:52,509] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 04:57:52,518] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 04:57:52,522] {logging_mixin.py:109} INFO - [2021-12-15 04:57:52,522] {dag.py:2914} INFO - Setting next_dagrun for tutorial_taskflow_api_etl to None
[2021-12-15 04:57:52,534] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py took 0.308 seconds
[2021-12-15 04:58:27,482] {processor.py:163} INFO - Started process (PID=64173) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 04:58:27,483] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py for tasks to queue
[2021-12-15 04:58:27,484] {logging_mixin.py:109} INFO - [2021-12-15 04:58:27,484] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 04:58:27,508] {processor.py:632} INFO - DAG(s) dict_keys(['tutorial_taskflow_api_etl']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 04:58:27,512] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 04:58:27,788] {logging_mixin.py:109} INFO - [2021-12-15 04:58:27,787] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 04:58:27,796] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 04:58:27,800] {logging_mixin.py:109} INFO - [2021-12-15 04:58:27,800] {dag.py:2914} INFO - Setting next_dagrun for tutorial_taskflow_api_etl to None
[2021-12-15 04:58:27,813] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py took 0.333 seconds
[2021-12-15 04:59:00,055] {processor.py:163} INFO - Started process (PID=64386) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 04:59:00,056] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py for tasks to queue
[2021-12-15 04:59:00,057] {logging_mixin.py:109} INFO - [2021-12-15 04:59:00,056] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 04:59:00,079] {processor.py:632} INFO - DAG(s) dict_keys(['tutorial_taskflow_api_etl']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 04:59:00,084] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 04:59:00,340] {logging_mixin.py:109} INFO - [2021-12-15 04:59:00,339] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 04:59:00,348] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 04:59:00,352] {logging_mixin.py:109} INFO - [2021-12-15 04:59:00,352] {dag.py:2914} INFO - Setting next_dagrun for tutorial_taskflow_api_etl to None
[2021-12-15 04:59:00,363] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py took 0.311 seconds
[2021-12-15 05:04:22,856] {processor.py:163} INFO - Started process (PID=891) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:04:22,857] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py for tasks to queue
[2021-12-15 05:04:22,857] {logging_mixin.py:109} INFO - [2021-12-15 05:04:22,857] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:04:22,880] {processor.py:632} INFO - DAG(s) dict_keys(['tutorial_taskflow_api_etl']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:04:22,885] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:04:23,148] {logging_mixin.py:109} INFO - [2021-12-15 05:04:23,148] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:04:23,157] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:04:23,162] {logging_mixin.py:109} INFO - [2021-12-15 05:04:23,161] {dag.py:2914} INFO - Setting next_dagrun for tutorial_taskflow_api_etl to None
[2021-12-15 05:04:23,174] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py took 0.321 seconds
[2021-12-15 05:04:58,104] {processor.py:163} INFO - Started process (PID=1055) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:04:58,105] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py for tasks to queue
[2021-12-15 05:04:58,105] {logging_mixin.py:109} INFO - [2021-12-15 05:04:58,105] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:04:58,128] {processor.py:632} INFO - DAG(s) dict_keys(['tutorial_taskflow_api_etl']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:04:58,133] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:04:58,395] {logging_mixin.py:109} INFO - [2021-12-15 05:04:58,395] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:04:58,403] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:04:58,408] {logging_mixin.py:109} INFO - [2021-12-15 05:04:58,408] {dag.py:2914} INFO - Setting next_dagrun for tutorial_taskflow_api_etl to None
[2021-12-15 05:04:58,420] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py took 0.319 seconds
[2021-12-15 05:05:32,179] {processor.py:163} INFO - Started process (PID=1355) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:05:32,180] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py for tasks to queue
[2021-12-15 05:05:32,180] {logging_mixin.py:109} INFO - [2021-12-15 05:05:32,180] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:05:32,203] {processor.py:632} INFO - DAG(s) dict_keys(['tutorial_taskflow_api_etl']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:05:32,207] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:05:32,457] {logging_mixin.py:109} INFO - [2021-12-15 05:05:32,457] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:05:32,465] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:05:32,469] {logging_mixin.py:109} INFO - [2021-12-15 05:05:32,469] {dag.py:2914} INFO - Setting next_dagrun for tutorial_taskflow_api_etl to None
[2021-12-15 05:05:32,480] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py took 0.305 seconds
[2021-12-15 05:06:06,761] {processor.py:163} INFO - Started process (PID=1475) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:06:06,762] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py for tasks to queue
[2021-12-15 05:06:06,763] {logging_mixin.py:109} INFO - [2021-12-15 05:06:06,763] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:06:06,785] {processor.py:632} INFO - DAG(s) dict_keys(['tutorial_taskflow_api_etl']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:06:06,790] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:06:07,030] {logging_mixin.py:109} INFO - [2021-12-15 05:06:07,030] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:06:07,038] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:06:07,043] {logging_mixin.py:109} INFO - [2021-12-15 05:06:07,043] {dag.py:2914} INFO - Setting next_dagrun for tutorial_taskflow_api_etl to None
[2021-12-15 05:06:07,053] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py took 0.295 seconds
[2021-12-15 05:06:40,765] {processor.py:163} INFO - Started process (PID=1616) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:06:40,766] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py for tasks to queue
[2021-12-15 05:06:40,766] {logging_mixin.py:109} INFO - [2021-12-15 05:06:40,766] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:06:40,793] {processor.py:632} INFO - DAG(s) dict_keys(['tutorial_taskflow_api_etl']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:06:40,797] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:06:41,066] {logging_mixin.py:109} INFO - [2021-12-15 05:06:41,066] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:06:41,074] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:06:41,079] {logging_mixin.py:109} INFO - [2021-12-15 05:06:41,078] {dag.py:2914} INFO - Setting next_dagrun for tutorial_taskflow_api_etl to None
[2021-12-15 05:06:41,091] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py took 0.330 seconds
[2021-12-15 05:07:14,800] {processor.py:163} INFO - Started process (PID=1711) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:07:14,801] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py for tasks to queue
[2021-12-15 05:07:14,802] {logging_mixin.py:109} INFO - [2021-12-15 05:07:14,802] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:07:14,825] {processor.py:632} INFO - DAG(s) dict_keys(['tutorial_taskflow_api_etl']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:07:14,830] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:07:15,086] {logging_mixin.py:109} INFO - [2021-12-15 05:07:15,086] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:07:15,094] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:07:15,098] {logging_mixin.py:109} INFO - [2021-12-15 05:07:15,098] {dag.py:2914} INFO - Setting next_dagrun for tutorial_taskflow_api_etl to None
[2021-12-15 05:07:15,111] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py took 0.314 seconds
[2021-12-15 05:07:49,658] {processor.py:163} INFO - Started process (PID=1796) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:07:49,659] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py for tasks to queue
[2021-12-15 05:07:49,659] {logging_mixin.py:109} INFO - [2021-12-15 05:07:49,659] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:07:49,682] {processor.py:632} INFO - DAG(s) dict_keys(['tutorial_taskflow_api_etl']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:07:49,686] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:07:49,932] {logging_mixin.py:109} INFO - [2021-12-15 05:07:49,932] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:07:49,940] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:07:49,944] {logging_mixin.py:109} INFO - [2021-12-15 05:07:49,944] {dag.py:2914} INFO - Setting next_dagrun for tutorial_taskflow_api_etl to None
[2021-12-15 05:07:49,957] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py took 0.303 seconds
[2021-12-15 05:08:23,675] {processor.py:163} INFO - Started process (PID=1880) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:08:23,675] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py for tasks to queue
[2021-12-15 05:08:23,676] {logging_mixin.py:109} INFO - [2021-12-15 05:08:23,676] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:08:23,699] {processor.py:632} INFO - DAG(s) dict_keys(['tutorial_taskflow_api_etl']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:08:23,703] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:08:23,970] {logging_mixin.py:109} INFO - [2021-12-15 05:08:23,970] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:08:23,979] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:08:23,985] {logging_mixin.py:109} INFO - [2021-12-15 05:08:23,985] {dag.py:2914} INFO - Setting next_dagrun for tutorial_taskflow_api_etl to None
[2021-12-15 05:08:23,998] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py took 0.327 seconds
[2021-12-15 05:08:58,926] {processor.py:163} INFO - Started process (PID=1988) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:08:58,926] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py for tasks to queue
[2021-12-15 05:08:58,927] {logging_mixin.py:109} INFO - [2021-12-15 05:08:58,927] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:08:58,952] {processor.py:632} INFO - DAG(s) dict_keys(['tutorial_taskflow_api_etl']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:08:58,956] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:08:59,207] {logging_mixin.py:109} INFO - [2021-12-15 05:08:59,207] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:08:59,217] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:08:59,221] {logging_mixin.py:109} INFO - [2021-12-15 05:08:59,221] {dag.py:2914} INFO - Setting next_dagrun for tutorial_taskflow_api_etl to None
[2021-12-15 05:08:59,232] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py took 0.310 seconds
[2021-12-15 05:09:34,587] {processor.py:163} INFO - Started process (PID=2212) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:09:34,588] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py for tasks to queue
[2021-12-15 05:09:34,588] {logging_mixin.py:109} INFO - [2021-12-15 05:09:34,588] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:09:34,612] {processor.py:632} INFO - DAG(s) dict_keys(['tutorial_taskflow_api_etl']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:09:34,616] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:09:34,874] {logging_mixin.py:109} INFO - [2021-12-15 05:09:34,874] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:09:34,884] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:09:34,888] {logging_mixin.py:109} INFO - [2021-12-15 05:09:34,888] {dag.py:2914} INFO - Setting next_dagrun for tutorial_taskflow_api_etl to None
[2021-12-15 05:09:34,900] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py took 0.316 seconds
[2021-12-15 05:10:10,995] {processor.py:163} INFO - Started process (PID=2394) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:10:10,996] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py for tasks to queue
[2021-12-15 05:10:10,997] {logging_mixin.py:109} INFO - [2021-12-15 05:10:10,997] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:10:11,023] {processor.py:632} INFO - DAG(s) dict_keys(['tutorial_taskflow_api_etl']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:10:11,028] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:10:11,286] {logging_mixin.py:109} INFO - [2021-12-15 05:10:11,286] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:10:11,296] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:10:11,300] {logging_mixin.py:109} INFO - [2021-12-15 05:10:11,300] {dag.py:2914} INFO - Setting next_dagrun for tutorial_taskflow_api_etl to None
[2021-12-15 05:10:11,312] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py took 0.320 seconds
[2021-12-15 05:10:48,742] {processor.py:163} INFO - Started process (PID=2649) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:10:48,743] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py for tasks to queue
[2021-12-15 05:10:48,743] {logging_mixin.py:109} INFO - [2021-12-15 05:10:48,743] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:10:48,766] {processor.py:632} INFO - DAG(s) dict_keys(['tutorial_taskflow_api_etl']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:10:48,770] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:10:49,043] {logging_mixin.py:109} INFO - [2021-12-15 05:10:49,042] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:10:49,052] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:10:49,056] {logging_mixin.py:109} INFO - [2021-12-15 05:10:49,056] {dag.py:2914} INFO - Setting next_dagrun for tutorial_taskflow_api_etl to None
[2021-12-15 05:10:49,073] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py took 0.334 seconds
[2021-12-15 05:11:20,478] {processor.py:163} INFO - Started process (PID=2900) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:11:20,479] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py for tasks to queue
[2021-12-15 05:11:20,480] {logging_mixin.py:109} INFO - [2021-12-15 05:11:20,480] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:11:20,503] {processor.py:632} INFO - DAG(s) dict_keys(['tutorial_taskflow_api_etl']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:11:20,507] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:11:20,763] {logging_mixin.py:109} INFO - [2021-12-15 05:11:20,763] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:11:20,771] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:11:20,776] {logging_mixin.py:109} INFO - [2021-12-15 05:11:20,776] {dag.py:2914} INFO - Setting next_dagrun for tutorial_taskflow_api_etl to None
[2021-12-15 05:11:20,787] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py took 0.312 seconds
[2021-12-15 05:11:53,068] {processor.py:163} INFO - Started process (PID=2985) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:11:53,068] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py for tasks to queue
[2021-12-15 05:11:53,069] {logging_mixin.py:109} INFO - [2021-12-15 05:11:53,069] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:11:53,092] {processor.py:632} INFO - DAG(s) dict_keys(['tutorial_taskflow_api_etl']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:11:53,097] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:11:53,371] {logging_mixin.py:109} INFO - [2021-12-15 05:11:53,370] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:11:53,380] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:11:53,384] {logging_mixin.py:109} INFO - [2021-12-15 05:11:53,384] {dag.py:2914} INFO - Setting next_dagrun for tutorial_taskflow_api_etl to None
[2021-12-15 05:11:53,396] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py took 0.331 seconds
[2021-12-15 05:12:26,522] {processor.py:163} INFO - Started process (PID=3102) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:12:26,523] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py for tasks to queue
[2021-12-15 05:12:26,523] {logging_mixin.py:109} INFO - [2021-12-15 05:12:26,523] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:12:26,546] {processor.py:632} INFO - DAG(s) dict_keys(['tutorial_taskflow_api_etl']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:12:26,550] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:12:26,810] {logging_mixin.py:109} INFO - [2021-12-15 05:12:26,810] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:12:26,818] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:12:26,822] {logging_mixin.py:109} INFO - [2021-12-15 05:12:26,822] {dag.py:2914} INFO - Setting next_dagrun for tutorial_taskflow_api_etl to None
[2021-12-15 05:12:26,833] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py took 0.315 seconds
[2021-12-15 05:13:01,453] {processor.py:163} INFO - Started process (PID=3276) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:13:01,454] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py for tasks to queue
[2021-12-15 05:13:01,455] {logging_mixin.py:109} INFO - [2021-12-15 05:13:01,455] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:13:01,478] {processor.py:632} INFO - DAG(s) dict_keys(['tutorial_taskflow_api_etl']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:13:01,483] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:13:01,746] {logging_mixin.py:109} INFO - [2021-12-15 05:13:01,746] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:13:01,755] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:13:01,761] {logging_mixin.py:109} INFO - [2021-12-15 05:13:01,761] {dag.py:2914} INFO - Setting next_dagrun for tutorial_taskflow_api_etl to None
[2021-12-15 05:13:01,772] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py took 0.322 seconds
[2021-12-15 05:13:35,786] {processor.py:163} INFO - Started process (PID=3392) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:13:35,787] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py for tasks to queue
[2021-12-15 05:13:35,787] {logging_mixin.py:109} INFO - [2021-12-15 05:13:35,787] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:13:35,810] {processor.py:632} INFO - DAG(s) dict_keys(['tutorial_taskflow_api_etl']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:13:35,815] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:13:36,076] {logging_mixin.py:109} INFO - [2021-12-15 05:13:36,076] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:13:36,085] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:13:36,090] {logging_mixin.py:109} INFO - [2021-12-15 05:13:36,090] {dag.py:2914} INFO - Setting next_dagrun for tutorial_taskflow_api_etl to None
[2021-12-15 05:13:36,102] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py took 0.319 seconds
[2021-12-15 05:14:11,863] {processor.py:163} INFO - Started process (PID=3540) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:14:11,864] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py for tasks to queue
[2021-12-15 05:14:11,864] {logging_mixin.py:109} INFO - [2021-12-15 05:14:11,864] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:14:11,888] {processor.py:632} INFO - DAG(s) dict_keys(['tutorial_taskflow_api_etl']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:14:11,893] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:14:12,160] {logging_mixin.py:109} INFO - [2021-12-15 05:14:12,159] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:14:12,168] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:14:12,172] {logging_mixin.py:109} INFO - [2021-12-15 05:14:12,172] {dag.py:2914} INFO - Setting next_dagrun for tutorial_taskflow_api_etl to None
[2021-12-15 05:14:12,184] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py took 0.324 seconds
[2021-12-15 05:14:44,174] {processor.py:163} INFO - Started process (PID=3775) to work on /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:14:44,175] {processor.py:620} INFO - Processing file /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py for tasks to queue
[2021-12-15 05:14:44,175] {logging_mixin.py:109} INFO - [2021-12-15 05:14:44,175] {dagbag.py:500} INFO - Filling up the DagBag from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:14:44,199] {processor.py:632} INFO - DAG(s) dict_keys(['tutorial_taskflow_api_etl']) retrieved from /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2021-12-15 05:14:44,203] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:14:44,460] {logging_mixin.py:109} INFO - [2021-12-15 05:14:44,460] {dag.py:2389} INFO - Sync 1 DAGs
[2021-12-15 05:14:44,468] {logging_mixin.py:109} WARNING - /opt/conda/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2021-12-15 05:14:44,473] {logging_mixin.py:109} INFO - [2021-12-15 05:14:44,472] {dag.py:2914} INFO - Setting next_dagrun for tutorial_taskflow_api_etl to None
[2021-12-15 05:14:44,484] {processor.py:171} INFO - Processing /opt/conda/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py took 0.313 seconds
